### ISSUE TITLE
ValueError: Namespace Notify not available for version 0.7

### ISSUE THREAD

**Issue by ghost** [lutris/lutris#4380]

**Comment by danieljohnson2** [comment]
We need a bit more than that. When does this happen?

Or maybe give us the debug output. Run 'lutris -d' and post the output when this error happens.

**Comment by SwetyCore** [comment]
> We need a bit more than that. When does this happen?
> 
> Or maybe give us the debug output. Run 'lutris -d' and post the output when this error happens.

problem has been resolved  by changing the required version in lutris
The problem should come from the system's libnotify version was upgraded from 0.7 to 0.8

**Comment by SwetyCore** [comment]
The problem reappeared today, still after I updated my ArchLinux system.
```
Traceback (most recent call last):
  File "/opt/apps/net.winegame.client/files/bin/lutris", line 52, in <module>
    from lutris.gui.application import Application  # pylint: disable=no-name-in-module
  File "/opt/apps/net.winegame.client/files/lib/lutris/lutris/gui/application.py", line 55, in <module>
    from .lutriswindow import LutrisWindow
  File "/opt/apps/net.winegame.client/files/lib/lutris/lutris/gui/lutriswindow.py", line 13, in <module>
    from lutris.game_actions import GameActions
  File "/opt/apps/net.winegame.client/files/lib/lutris/lutris/game_actions.py", line 13, in <module>
    from lutris.gui.config.add_game import AddGameDialog
  File "/opt/apps/net.winegame.client/files/lib/lutris/lutris/gui/config/add_game.py", line 4, in <module>
    from lutris.gui.config.common import GameDialogCommon
  File "/opt/apps/net.winegame.client/files/lib/lutris/lutris/gui/config/common.py", line 16, in <module>
    from lutris.gui.widgets.notifications import send_notification
  File "/opt/apps/net.winegame.client/files/lib/lutris/lutris/gui/widgets/notifications.py", line 7, in <module>
    gi.require_version('Notify', '0.8')
  File "/usr/lib/python3.10/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Notify not available for version 0.8

```
